janusgraph-core/src/main/java/org/janusgraph/core/Transaction.java

```diff
@@ -68,7 +68,7 @@
      * Returns a {@link org.janusgraph.core.JanusGraphIndexQuery} to query for vertices or edges against the specified indexing backend using
      * the given query string. The query string is analyzed and answered by the underlying storage backend.
      * <p>
-     * Note, that using indexQuery will may ignore modifications in the current transaction.
```

Comment:
While it is an improvement grammatically I actually disagree with @pluradj on this one. I believe it should be that using indexQuery **will** ignore modifications in the current transaction. 

If you're doing a direct query against the index any changes made in the current open transaction will not be written yet, and therefore not available. I will yield and approve if Jason still thinks **may** is more appropriate.

Reply:
This comment has been in place going back to Titan. I haven't dug into the code to verify whether "will" or "may" is correct. My impression was that there could be some doubt, perhaps given different characteristics of the chosen storage backend, so that's why "may" might make more sense.